src/amp-story-player/amp-story-player-impl.js

```diff
@@ -1546,6 +1569,8 @@ export class AmpStoryPlayer {
     this.pageScroller_ &&
       this.pageScroller_.onTouchStart(event.timeStamp, coordinates.clientY);
 
+    this.rootEl_.classList.remove(CLASS_NO_NAVIGATION_TRANSITION);
```

Comment:
I think we should find a better place to remove this class. 

There will be no animation if e.g.:
1. User navigates with a button to a different story that uses `go(1, null, {animate: false})`
2. User taps through a story and player navigates to the next story.
3. Expected: an animation will show
4. Actual: no animation is shown

Reply:
Ok so to streamline thoughts, these are the places that need to remove this class:
 1. On story end (tap to advance to the next story or autoadvance)
 2. Navigation swipe (this place)

Any other places where these animations will happen?

My guess is that removing this class on those two cases will work well, as I'm not sure there's a unique place where we can tackle both cases at once. I'll double check this to be sure.

Reply:
Yeah, ideally we would want to have this in as few places as possible to avoid forgetting to remove it in case we add another navigation mechanism.

An idea that comes to mind is waiting for the `renderPromise` inside `show()` to complete and then remove it.

Reply:
Seems like `renderPromise` doesn't have anything to do with the timing of transitions, but the event `transitionend`  (which bubbles to `rootEl_`) can be used to remove the class if needed